tests/FSharp.TestHelpers/Utilities.fs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
```

Comment:
Is there still a Utilities.fs in **tests/fsharp/Compiler/** ?  I wonder why it wasn't a rename?

Reply:
No, I've moved the file, also wondering why doesn't git show it as rename. I've git mv'd it.

Reply:
Really weird, the first commit showed just rename:
![image](https://user-images.githubusercontent.com/1260985/83364916-04949880-a3a5-11ea-8da0-70efcee59b32.png)